sanic/cli/arguments.py

```diff
@@ -277,6 +277,15 @@ def attach(self):
                 "changes"
             ),
         )
+        self.container.add_argument(
+            "-i",
+            "--interval",
+            "--auto-reload-interval",
+            dest="auto_reload_interval",
+            default=1.0,
+            type=float,
+            help="Interval in seconds to check for file changes [default 1.0]",
+        )
```

Comment:
Nice. We will need to add a test for this.

Reply:
![image](https://user-images.githubusercontent.com/166269/195154336-eea34a96-cd54-413f-8869-f2a03cf34fc8.png)

Since you are adding more support for this feature, we should probably also add it to the MOTD right here:

```
auto-reload: enabled [1.0s]
```